From 190f31d2a5a1b5d852fff108258ab5265fd35e07 Mon Sep 17 00:00:00 2001
From: Kai Tietz <ktietz@redhat.com>
Date: Mon, 25 Mar 2013 15:52:48 +0100
Subject: Backport of SVN 197049

 Patch edited by Tim S.
 2013-03-25  Richard Biener  <rguenther@suse.de>

        * config/i386/xm-mingw32.h (__USE_MINGW_ANSI_STDIO): Enable
        POSIX-printf for mingw-hosted  builds.

---
 gcc/config/i386/xm-mingw32.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/gcc/config/i386/xm-mingw32.h b/gcc/config/i386/xm-mingw32.h
index e0dd3f37211..b6edde5aead 100644
--- a/gcc/config/i386/xm-mingw32.h
+++ b/gcc/config/i386/xm-mingw32.h
@@ -30,6 +30,11 @@ along with GCC; see the file COPYING3.  If not see
 /*  The st_ino field of struct stat is always 0.  */
 #define HOST_LACKS_INODE_NUMBERS
 
+#ifdef __MINGW32__
+#undef __USE_MINGW_ANSI_STDIO
+#define __USE_MINGW_ANSI_STDIO 1
+#else
 /* MSVCRT does not support the "ll" format specifier for printing
    "long long" values.  Instead, we use "I64".  */
 #define HOST_LONG_LONG_FORMAT "I64"
+#endif
-- 
